runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -200,20 +205,25 @@ public void popupMenuCanceled(PopupMenuEvent popupMenuEvent)
 		statsPanel.add(bottomLeftStat);  // bottom left
 		statsPanel.add(bottomRightStat); // bottom right
 
-		headerPanel.add(skillIcon, BorderLayout.WEST);
+		JLabel headerSkillIcon = getSkillIcon(iconManager, skill, 35, 35, false);
+		headerPanel.add(headerSkillIcon, BorderLayout.WEST);
 		headerPanel.add(statsPanel, BorderLayout.CENTER);
 
-		JPanel progressWrapper = new JPanel();
 		progressWrapper.setBackground(ColorScheme.DARKER_GRAY_COLOR);
 		progressWrapper.setLayout(new BorderLayout());
-		progressWrapper.setBorder(new EmptyBorder(0, 7, 7, 7));
+		progressWrapper.setBorder(DEFAULT_PROGRESS_WRAPPER_BORDER);
 
 		progressBar.setMaximumValue(100);
 		progressBar.setBackground(new Color(61, 56, 49));
 		progressBar.setForeground(SkillColor.find(skill).getColor());
 		progressBar.setDimmedText("Paused");
 
-		progressWrapper.add(progressBar, BorderLayout.NORTH);
+		JLabel progressSkillIcon = getSkillIcon(iconManager, skill, 10, 10, true);
```

Comment:
Instead of using the full size skill images here, [use `SkillIconManager`'s `small` argument to get a smaller image to work with](http://static.runelite.net/api/runelite-client/net/runelite/client/game/SkillIconManager.html#getSkillImage(net.runelite.api.Skill,boolean)). Even if the icon needs to be further resized, this will be a better starting point for smaller icons.